IdentityCore/src/validation/MSIDAuthority.h

```diff
@@ -110,4 +110,6 @@ typedef void(^MSIDOpenIdConfigurationInfoBlock)(MSIDOpenIdProviderMetadata * _Nu
                        context:(nullable id<MSIDRequestContext>)context
                          error:(NSError * _Nullable __autoreleasing * _Nullable)error;
 
+- (BOOL)needsUpdateToHomeAuthority:(BOOL)isAccountFromMSATenant;
```

Comment:
Can you add a comment to describe the purpose of this API? Unclear when it should be used and maybe it can be helpful to someone for other purposes.